update readme for archiving (#39)
fortinet/fortigate-autoscale-aws-tgw: README.md

# FortiGate Autoscale with AWS Transit Gateway Integration

This repository has been archived.

> The ***FortiGate Autoscale with AWS Transit Gateway Integration*** project has been replaced by [https://github.com/fortinet/fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws). The Transit Gateway integration feature provided here in this project is also available in the new [fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws) project.

Reviewer: ```suggestion
The ***FortiGate Autoscale with AWS Transit Gateway Integration*** project has been moved to a new location: [https://github.com/fortinet/fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws).
```

Reviewer: When users click the URL, they will see Transit Gateway mentioned in the About:
AWS Transit Gateway can be used to connect Amazon Virtual Private Clouds (VPCs) and their on-premises networks to a single gateway. FortiGate Autoscale with Transit Gateway integration extends the protection to all networks connected to the Transit Gateway.
